client/client.go

```diff
@@ -109,11 +109,11 @@ func New(opts ...Option) (*Client, error) {
 	return c, nil
 }
 
-// Export takes a Jaeger batch and uses one of the available workers to export it synchronously.
+// Export takes a Jaeger batches and uses one of the available workers to export it synchronously.
 // It returns an error in case a request cannot be processed. It's up to the caller to retry.
-func (sa *Client) Export(ctx context.Context, batch *jaegerpb.Batch) error {
+func (sa *Client) Export(ctx context.Context, batches []*jaegerpb.Batch) error {
```

Comment:
This is a breaking change in a public function. Do we import this repo using a specific commit?
Perhaps to be safe just add an `ExportMultiple` function and keep the old one for now?

Reply:
The package is versioned and is fixed in the mod file so we can make a breaking change without breaking anyone's builds. Let's make sure to bump the major version to be clear.

Reply:
Sure. we'll release it as 0.5